### ISSUE TITLE
Funktiot

### ISSUE THREAD

**Issue by jukop** [jukop/r_intro#4]
Funktioiden lyhytmuoto puuttuu verkkoluentomonisteesta. Rcourse -paketti käyttää sitä kuitenkin paljon.

**Comment by jukop** [comment]
Extra: funktio ifelse (samoin kuin case_when) vaativat että palautetaan objekti, joka on samaa tyyppiä. R-kielen rakenne if-elseif-else mahdollistaa erityyppisten objektien palauttamisen (funktio ei).

**Comment by santikka** [comment]
Totta, ifelse pitäisi ehkä opettaa "vektorin täyttäjänä", jossa täyttö jokaisen alkion kohdalla tehdään ehdon perusteella. Koska tulos on vektori, niin myös vaihtoehtojen on oltava samaa tyyppiä (tai vähintäänkin muunnettavissa samaksi). Harvemminen tätä näkee oikeiden ehtolausekkeiden korvaajana.

**Comment by santikka** [comment]
Lisäsin kappaleen lyhytmuodosta.

**Comment by jukop** [comment]
Tarvittaisiin Miten ohjelmoidaan funktio -video.
